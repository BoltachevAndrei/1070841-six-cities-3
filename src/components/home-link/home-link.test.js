import React from 'react';
import {Router} from 'react-router-dom';
import history from '../../history.js';
import renderer from 'react-test-renderer';
import HomeLink from './home-link.jsx';

it(`HomeLink renders correctly`, () => {
  const tree = renderer
    .create(
        <Router
          history={history}
        >
          <HomeLink />
        </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
