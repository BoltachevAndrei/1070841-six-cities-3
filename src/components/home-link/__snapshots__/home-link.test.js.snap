// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeLink renders correctly 1`] = `
<a
  className="header__logo-link"
  href="/"
  onClick={[Function]}
>
  <img
    alt="6 cities logo"
    className="header__logo"
    height="41"
    src="img/logo.svg"
    width="81"
  />
</a>
`;
